app/components/landing/InfoCard.tsx

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable max-len */
+import React, { useState } from 'react';
+import {
+    Card,
+} from 'antd';
+import DiscordButton from '../common/DiscordButton';
+
+type InfoTab = 'welcome'|'howDoesThisWork'|'getInvolved';
+
+const InfoCard = (): JSX.Element => {
+    const [infoTab, setInfoTab] = useState<InfoTab>('welcome');
+    const tabList = [
+        {
+            key: 'welcome',
+            tab: 'What\'s this?',
+        },
+        {
+            key: 'howDoIUseThis',
+            tab: 'How do I use this?',
+        },
+        {
+            key: 'howDoesThisWork',
+            tab: 'How does it work?',
+        },
+        {
+            key: 'getInvolved',
+            tab: 'Get involved!',
+        },
+    ];
+
+    const welcomeText = [
+        'This is TMDojo - our platform to analyze Trackmania runs!',
+        '',
+        'Have you ever wondered how the WR drove that insane first turn?',
+        'Ever been confused because you just can\'t figure out that one dirt gear?',
+        '',
+        'We want to give you the tools to really understand your replays - with a 3D view of the map and a ton of data!',
+    ];
+    const howDoIUseThisText = [
+        'Simply pick a map from the table and open it.',
+        '',
+        'In the map view, open the replay menu on the left and load the replays you want to inspect.',
+        'Now you can change the display settings to different modes using the menu on the right.',
+        '',
+        'To navigate around the 3D view, use right mouse button to move and left to rotate - scroll wheel zooms in and out.',
+    ];
+    const howDoesThisWorkText = [
+        'We\'re planning on supporting several ways for you to submit runs:',
+        '1. If you\'re using Openplanet, you can use our plugin to automatically upload your runs - only your PBs if you want.',
+        '2. You can also upload a replay yourself, and we\'ll extract all the useful data.',
+        '3. To make it even easier, we might also support fetching a time straight off the leaderboards - so you don\'t have to download it yourself.',
+        '',
+        'At the moment we\'re not ready for everyone\'s runs yet.',
+        'If you\'re interested in testing stuff for us in the future, check out \'Get involved\'!',
+    ];
+    const getInvolvedText = [
+        'TMDojo is still very much in development!',
+        '',
+        'We\'re working on it in our free time, so updates will come when they\'re ready.',
+        '',
+        <span>
+            <span>In the meantime, feel free to join our </span>
+            <DiscordButton />
+            <span> and let us know what you think!</span>
+        </span>,
+    ];
+    const tabContent = {
+        welcome: (
+            <div>{welcomeText.map((text, i) => <p key={`${text}_${i}`}>{text === '' ? <br /> : text}</p>)}</div>
```

Comment:
You could write a small wrapper for these `.map` calls as they are the same for all tabs